components/carbon-authenticators/thrift-authenticator/org.wso2.carbon.identity.authenticator.thrift/src/main/resources/thrift-authentication.xml

```diff
@@ -41,4 +41,8 @@
     <!--30 min-->
     <ThriftSessionTimeout>1800000</ThriftSessionTimeout>
 
+    <SSLEnabledProtocols>TLSv1,TLSv1.1,TLSv1.2</SSLEnabledProtocols>
```

Comment:
Let's not add these configurations by default. We should include this recommendation to https://docs.wso2.com/display/ADMIN44x/Configuring+Transport+Level+Security